src/cogent3/core/alignment.py

```diff
@@ -73,6 +73,7 @@
     extend_docstring_from,
     get_format_suffixes,
     get_object_provenance,
+    open_,
```

Comment:
I think we should be using the `atomic_write` class instead for writing to files, so import that class instead.